client/src/components/Input.js

```diff
@@ -1,12 +1,20 @@
 import styled from '@emotion/styled';
 import colors from '../utils/colorsUtils';
 
+const inputSize = {
+  large: { padding: '10px' },
+};
+
+const getInputPadding = (size = 'large') => {
+  return inputSize[size].padding;
+};
+
 const Input = styled.input`
   background-color: ${colors.primaryInputBackground};
   color: ${colors.primaryFont};
   font-size: 20px;
   border-radius: 14px;
-  padding-left: 15px;
+  padding: ${(props) => getInputPadding(props.size)};
```

Comment:
Good job 👍 
A function is fine here. If you want to save some lines of code, you could directly access `inputSize`.
 
`padding: ${(props) =>inputSize[props.size].padding;`
If `size` is optional, this line would look like this:
`padding: ${(props) =>inputSize[props.size || 'large'].padding;`

Still, it's not required to change anything here 👍